src/pages/stage-page/StagePage.tsx

```diff
@@ -15,6 +16,7 @@ export interface StagePageProps {
 }
 
 export const StagePage = ({
+  stageName,
```

Comment:
FYI, this has caused a type error in `stages.tsx`

Reply:
can we use the title prop instead of adding stageName?

Reply:
There are spaces in the title but I can swap them out for dashes